devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy

```diff
@@ -16,7 +16,6 @@ metadata {
 	definition(name: "Z-Wave Motion Light", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mmnm: "SmartThings", vid: "generic-motion-light") {
```

Comment:
this metadata still has `illuminance`

Reply:
No, it doesn't. I've removed it in MR 1017

Reply:
@PKacprowiczS You will need to request the metadata to production in the portal (I can't do that as of now, unfortunately). That will publish it in to acceptance. Technically we are supposed to only publish to acceptance on Tuesdays. I'll go ahead and merge this PR. I suppose that we can just request the metadata to acceptance (or, just do that next Tuesday which will line up with this PR getting deployed to staging and acceptance) before hand. I'll deploy the metadata to production when this is deployed to production.